src/controllers/EventsController.js

```diff
@@ -39,7 +40,7 @@ class EventsController extends Controller {
       this.response.write(`event: ${event.name}\n`);
       this.response.write(`data: ${JSON.stringify(eventToApi(event))}\n\n`);
     } catch (error) {
-      logger.error(`pipeEvents - write error: ${error}`);
+      logger.error({ err: error }, 'pipeEvents - write error');
```

Comment:
there should be a way to make `error` prop parsed as Error and use instead `err`